hubblestack/files/hubblestack_nova/misc.py

```diff
@@ -400,55 +410,40 @@ def check_path_integrity(reason=''):
     """
     Ensure that system PATH variable is not malformed.
     """
+    path_value = os.environ.get('PATH')
+    output = ''
+    for item in path_value.split(':'):
```

Comment:
if `path_value` is `None`, which is the default value should the `'PATH'` key not be present in `os.environ`, this call will fail. Maybe ensure an appropriate default value is given?